src/encoded/static/components/app.js

```diff
@@ -1123,6 +1196,12 @@ class App extends React.Component {
                                     </div>
                                 </Provider>
                             </div>
+                            {this.state.eulaModalVisibility ?
+                                <EulaModal
+                                    closeModal={this.closeSignupModal}
+                                    signup={() => this.signup(this.state.authResult)}
```

Comment:
Is there a benefit to using an inline function to call an internal method, instead of just calling the internal method directly?